integration/v3_grpc_test.go

```diff
@@ -491,3 +491,168 @@ func TestV3WatchCancel(t *testing.T) {
 
 	clus.Terminate(t)
 }
+
+func TestV3RangeRequest(t *testing.T) {
+	tests := []struct {
+		putKeys []string
+		reqs    []pb.RangeRequest
+
+		wresps [][]string
```

Comment:
how about change these to `wresps []pb.RangeResponse`? Then we can also test the fields in header like revision.